apps/central/src/components/admin/Admin.vue

```diff
@@ -57,13 +48,17 @@ export default {
     return {
       loading: false,
       error: null,
-      showSigninForm: true,
     };
   },
   computed: {
     selected() {
       return this.$route.name;
     },
   },
+  created() {
```

Comment:
we use .../admin as a way to login with username and password on a server that has OIDC login turned on , this would break that feature

Reply:
So how would you suggest we solve this? As far as I understand the current implementation we don't get a role on the session, which makes it kind of hard to do this differently...